src/helm/proxy/clients/http_client.py

```diff
@@ -0,0 +1,141 @@
+from dataclasses import asdict
+from typing import Optional
+
+from helm.common.cache import Cache, CacheConfig
+from helm.common.request import (
+    Request,
+    RequestResult,
+    Sequence,
+    Token,
+    EMBEDDING_UNAVAILABLE_REQUEST_RESULT,
+)
+from helm.common.tokenization_request import (
+    DecodeRequest,
+    DecodeRequestResult,
+    TokenizationRequest,
+    TokenizationRequestResult,
+    TokenizationToken,
+)
+from .client import Client, wrap_request_time
+
+import requests
+
+
+class HTTPModelClient(Client):
+    """Implements a simple client for a model being served over HTTP."""
+
+    def __init__(
+        self,
+        cache_config: CacheConfig,
+        base_url: str = "http://localhost:8080",
+        timeout: int = 10,
+        do_cache: bool = False,
+    ):
+        self.do_cache = do_cache
+        self.cache: Optional[Cache] = Cache(cache_config) if do_cache else None
+        self.base_url = base_url
+        self.timeout = timeout
+
+    def make_request(self, request: Request) -> RequestResult:
+        cache_key = asdict(request)
+        # This needs to match whatever we define in pedantic
+        if request.embedding:
+            return EMBEDDING_UNAVAILABLE_REQUEST_RESULT
+
+        raw_request = {
+            "prompt": request.prompt,
+            "temperature": request.temperature,
```

Comment:
temperature might be set to zero here which can lead to `inf` tensor [error](https://github.com/Lightning-AI/lit-gpt/issues/401).

Maybe a good idea to add `1e-7` like the HF client [here](https://github.com/stanford-crfm/helm/blob/de02a1e05fa438d718f9a22dc5cff40f33f8ee41/src/helm/proxy/clients/huggingface_client.py#L188)?

Reply:
Got it, changed back